controller/auth-controller.js

```diff
@@ -0,0 +1,66 @@
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+const User = require("../models/user");
+
+exports.login = async (req, res) => {
+  const name = req.body.name;
+  const pass = req.body.pass;
```

Comment:
Assign contents of `req.body` in every function like this - 
`const { name, pass } = req.body`

Reply:
done